tsconfig.vitest.json

```diff
@@ -1,6 +1,6 @@
 {
   "extends": "@vue/tsconfig/tsconfig.node.json",
-  "include": ["src/**/__tests__/*"],
+  "include": ["src/*"],
```

Comment:
Why do we change it?

Reply:
Updated this to fix the linting error on my system.